spec/dummy/app/avo/resources/spouse_resource.rb

```diff
@@ -9,4 +9,34 @@ class SpouseResource < Avo::BaseResource
 
   field :id, as: :id
   field :name, as: :text
+
+  # Uncomment this to test visibility inside tabs, panels and sidebar
+  # tabs do
+  #   field :hidden_field_inside_tabs, as: :text, visible: -> (resource:) { false }
+
+  #   tab "hidden testing visibility inside tabs" do
+  #     field :hidden_field_inside_tabs_inside_tab, as: :text, visible: -> (resource:) { false }
```

Comment:
This is still visible on my end.

Reply:
did you restart the local server before trying this one?